src/hclust.jl

```diff
@@ -509,7 +509,7 @@ end
 
 ## reorders the tree merges by the height of resulting trees
 ## (to be compatible with R's hclust())
-function rorder!(hmer::HclustMerges)
+function orderleaves_r!(hmer::HclustMerges)
```

Comment:
Before we merge this: should we rather call these methods `orderbranches_xxx!`?
This sounds more correct to me. Sorry for extra work.

Reply:
While it *does* order the branches, in my mind the important part is the order of the leaves. Ordering of the branches is incidental. But also I don't really feel strongly about it. Would you want to change the arg in `hclust()` to `branchorder=:r` ?

Reply:
> Would you want to change the arg in hclust() to branchorder=:r ?

Yes, it should be in sync.